mkosi/config.py

```diff
@@ -575,6 +575,7 @@ class ArtifactOutput(StrEnum):
     pcrs = enum.auto()
     roothash = enum.auto()
     os_release = enum.auto()
+    metainfo = enum.auto()
```

Comment:
that doesn't match the documentation anymore
```suggestion
    appstream_metainfo = enum.auto()
```